MMM-MotionDetector.js

```diff
@@ -68,35 +73,54 @@ Module.register("MMM-MotionDetector", {
       video: video,
       captureIntervalTime: this.config.captureIntervalTime,
       motionCanvas: canvas,
+
       initSuccessCallback: () => {
         Log.info("MMM-MotionDetector: DiffCamEngine init successful.");
+        if (this.config.debug) {
```

Comment:
you dont need that debug option. please just use Log.info for that logging. anyone who doesnt want those infos can change the log level in the MM core

Reply:
Done.